.github/CODEOWNERS

```diff
@@ -2,7 +2,30 @@
 # see https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 # for further details
 
+
```

Comment:
Curious to hear your thoughts on adding code ownership by pods too? For example, I would like to be involved in reviews on code changes on `/admin` especially since we're at the stage of cleaning up and establishing best practices.

Reply:
Also, I definitely want to look as CodeOwners as something we tweak and iterate towards a solution.